test_lib/compaction.py

```diff
@@ -7,6 +7,21 @@
 LOGGER = logging.getLogger(__name__)
 
 
+class GcMode(Enum):
+    REPAIR = "repair"
+    DISABLED = "disabled"
+    TIMEOUT = "timeout"
+    IMMEDIATE = "immediate"
+
+    @classmethod
+    def from_str(cls, output_str):
```

Comment:
looks this is not used. Is it really needed? see example code:
```
>>> from enum import Enum
>>> class en(Enum):
...   R = 'rep'
...   B = "ber"
... 
>>> en.R
<en.R: 'rep'>
>>> en('rep')
<en.R: 'rep'>

```

Reply:
yes, it was initially actually used, but postponed until https://github.com/scylladb/scylla/issues/10309 is fixed.

Reply:
@scylladb/qa-maintainers might prefer to remove unused code, and to re-introduce them once the issue is fixed.

Reply:
ok i commented out this method temporarily and added a TODO.